airflow/utils/context_to_airflow_vars.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+def context_to_airflow_vars(context):
```

Comment:
I was mainly -1 on the filename like analysis.py (since the functionality is more generic). I am OK with the more generic operator_helpers too, as long as it doesn't grow into another models.py. Ideally there would be a context class and then this method could be placed in there.